src/app/replayer/replayer.ml

```diff
@@ -1311,8 +1313,61 @@ let main ~input_file ~output_file_opt ~archive_uri ~continue_on_error
                     ] ;
                 Deferred.unit )
               else
+                let%bind accounts_before = Ledger.to_list ledger in
```

Comment:
I this an expensive operation? May it make sense to perform the added operations only in migration mode (as the results os this seem to be discarded in non-migration mode)